bin/test/errors/parsing-errors.t

```diff
@@ -112,7 +112,7 @@ Mismatch between section-start and section-end:
   $ mustache foo.json $PROBLEM
   Template parse error:
   File "wrong-nesting.mustache", lines 1-2, characters 41-0:
-  Section mismatch: {{#foo}} is closed by {{/bar}}.
+  Open/close tag mismatch: {{# foo }} is closed by {{/ bar }}.
```

Comment:
Does this style not apply to variables? E.g.  `{{$ foo }}..{{/ foo}}`

Reply:
It does, I just added test cases for `{{$foo}}` and `{{<foo}}`. Feel free to recommend improvements to the wording.

Reply:
Sorry, what I meant was that `pp_partial_param` isn't consistent with this printing style. It's still printing `{{$foo}}` instead of `{{$ foo }}`.